docs/operators/filter.md

```diff
@@ -0,0 +1,49 @@
+Filter
+=============
+
+## Operator Objective
+This operator receives an POJO ([Plain Old Java Object](https://en.wikipedia.org/wiki/Plain_Old_Java_Object)) as an incoming tuple
+and based on the filter condition it shall emit the filtered POJO on one output port and the rest on another output port.
+
+Filter operator supports quasi Java expressions to specify filter rule.
+
+Filter operator does not hold any state and is **idempotent**, **fault-tolerant** and **statically/dynamically partitionable**.
+
+## Operator Usecase
+1. ***Customer data*** usually contains a field customer category/segment. One wants some analysis to be done for specific customer segment. One could filter the records based on segment.
+2. ***Log data*** processing pipeline may want to filter logs from specific machine/router/switch.
+
+## Operator Information
+1. Operator location: ***malhar-library***
```

Comment:
Would it make sense to make this a link to : https://github.com/apache/apex-malhar/tree/master/library